src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,30 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistrbucion(val ciudad: Ciudad){
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (vendedores.contains(vendedor)){
+            throw Exception("Ya está registrado en el centro")
+        }
+        else{
+            vendedores.add(vendedor)
+        }
+    }
+
+    fun vendedorEstrella(): Vendedor? {
+        return vendedores.maxBy { v -> v.puntajeCertificaciones() }
+    }
```

Comment:
En la materia vamos a insistir con que usen el `=` para definir consultas (métodos que devuelven algo) y dejen las llaves para las acciones (métodos que no devuelven nada). De esta forma, además te ahorrás escribir el tipo de retorno.

```suggestion
    fun vendedorEstrella() = vendedores.maxBy { v -> v.puntajeCertificaciones() }
```